src/Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.csproj

```diff
@@ -10,6 +10,6 @@
     <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <IncludeSymbols>true</IncludeSymbols>
-    <IsShippingAssembly>true</IsShippingAssembly>
+    <IsShipping>true</IsShipping>
```

Comment:
Why is IsShipping set here and the rest set IsShippingPackage to false?

Reply:
All the other places have shipping assemblies (default) but not shipping packages. This one both the package and the assemblies ship.